contracts/bonding/BondingManager.sol

```diff
@@ -1089,6 +1089,14 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
      */
     function updateDelegatorWithEarnings(address _delegator, uint256 _endRound) internal {
         Delegator storage del = delegators[_delegator];
+        uint256 lastClaimRound = del.lastClaimRound;
+
+        // Return if autoClaimEarnings is executed multiple times in a single round
+        if (lastClaimRound >= _endRound) return;
```

Comment:
Yeah that's fine by me. The reason I tried doing it the other way is because it saves an `SLOAD` operation when the conditional would pass.  If we do the check in `autoClaimEarnings` we're making an additional SLOAD to get the `lastClaimRound` for `msg.sender`. 

Fixed in 97dd687

Reply:
Re: the additional SLOAD - what if we added a `uint256 _lastClaimRound` param to `updateDelegatorWithEarnings()` and then the function caller (i.e. `autoClaimEarnings()` and `claimEarnings()`) would read the `lastClaimRound` for a delegator from storage and pass it in to `updateDelegatorWithEarnings()` as an argument?

Reply:
Yeah I think that would work.

Loading a storage pointer  (`Delegator storage del = delegators[_delegator];
`) is a `PUSH` and not `SLOAD` , right?

Reply:
> Loading a storage pointer (Delegator storage del = delegators[_delegator]; ) is a PUSH and not SLOAD , right?

I believe that should be the case.

> I didn't really want to modify the API (even though it's an internal function). But if you're good with it , let's do it.

In this case, I'm fine with a small change to the API of an internal function since the function's usage is isolated in two areas.

Reply:
fixed in 6e24b69